### PR TITLE
feat(cdk): Resize the volume

### PR DESCRIPTION
**Pull request by seebees**
As part of the bootstrap just resize the volume.

Related: #166 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mattsb42-aws** [comment]
This needs to happen before anything else. `make bootstrap` fails every time I run it now because the disk fills up: #173

File: cdk/bin/resize.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Resize the EBS volume.
+aws ec2 modify-volume --volume-id $VOLUMEID --size $SIZE
+
+# Wait for the resize to finish.
+while [ "$(aws ec2 describe-volumes-modifications --volume-id $VOLUMEID --filters Name=modification-state,Values="optimizing","completed" | jq '.VolumesModifications | length')" != "1" ]; do
+  sleep 1
+  done
+
+# Rewrite the partition table so that the partition takes up all the space that it can.
+sudo growpart /dev/nvme0n1p1 1
```
Comment (mattsb42-aws) [inline cdk/bin/resize.sh]: This fails when I try to run it due to:

```
~/environment/workshop (master) $ sudo growpart /dev/nvme0n1p1 1
FAILED: unable to determine partition type

```

Comment (mattsb42-aws) [reply cdk/bin/resize.sh]: It seems like just waiting for the EBS volume to finish resizing, running resize2fs, and rebooting worked.

Comment (mattsb42-aws) [reply cdk/bin/resize.sh]: Could we just do an end-run around all of this and make the original Cloud 9 instance create with a bigger volume?

Comment (seebees) [reply cdk/bin/resize.sh]: Looked more at: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/recognize-expanded-volume-linux.html

I think that the partition was not being referenced correctly.

**Comment by mattsb42-aws** [comment]
I really think this needs to run first, and by itself. Letting it run alongside everything else means that we *will inevitably* hit a race condition where one of the other things fills up the disk before the resize finishes, or places it in some other condition where we run into a conflict.

The bootstrap target already takes a long time to run. Adding on another 10-15s to avoid nondeterministic breakage is fine.

**Comment by seebees** [comment]
You are right.  Race removed.

**Pull request merged**